docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
 
 For detailed instructions on each step, follow the links above.
 
-## How do I get access to the private channel with Icebreaker documentation?
+### What endpoints do I need to open in my firewall for the Icebreaker node?
 
-After receiving your Icebreaker NFT:
+To function properly, your Icebreaker node requires the following ports to be open in your firewall:
 
-- Notify the Blockfrost team that you've received your NFT
-- A team member will add you to the private Icebreakers channel
-- Access the comprehensive documentation in the announcement channel
-- Follow the setup instructions provided there.
+- `3000/tcp` (or custom port): For incoming API requests from Blockfrost
+- `8081/tcp`: Prometheus metrics endpoint for monitoring (optional but recommended)
```

Comment:
and metrics can be disabled all together.

Reply:
I added that detail.